### PR TITLE
Ticket29299 squashed rebased

### PR DESCRIPTION
**Pull request by juga0**

### CODE REVIEW

File: docs/source/man_sbws.ini.rst [hunk 1]
```diff
@@ -72,6 +72,9 @@ destinations.STR
   verify = BOOL
     Whether or not to verify the destination certificate.
     (Default: True)
+  country = STR
+    ISO 3166-1 alpha-2 country code.
+    Use 00 if the destination URL is a domain name and it is in a CDN.
```
Comment (tlyu) [inline docs/source/man_sbws.ini.rst]: Maybe use another alphabetic user-assigned code like `ZZ`? It seems odd to use numeric `00` here when there are still some user-assigned codes available.

File: sbws/core/scanner.py [hunk 2]
```diff
@@ -557,7 +557,6 @@ def gen_parser(sub):
 def main(args, conf):
     if conf.getint('scanner', 'measurement_threads') < 1:
         fail_hard('Number of measurement threads must be larger than 1')
-
```
Comment (tlyu) [inline sbws/core/scanner.py]: Please don't include formatting changes in a commit that contains code changes. If this reformatting needs to happen, it should go in a separate commit.

File: docs/source/man_sbws.ini.rst (hunk 1 above)
Comment (juga0) [reply docs/source/man_sbws.ini.rst]: Changed to ZZ

File: sbws/core/scanner.py (hunk 2 above)
Comment (juga0) [reply sbws/core/scanner.py]: Was a mistake. Fixed

File: sbws/core/scanner.py [hunk 3]
```diff
@@ -571,5 +571,4 @@ def main(args, conf):
     # Generate an unique identifier for each scanner
     if 'uuid' not in state:
         state['uuid'] = str(uuid.uuid4())
-
```
Comment (tlyu) [inline sbws/core/scanner.py]: This is one more extraneous minor format change.

Comment (juga0) [reply sbws/core/scanner.py]: ok, fixup to solve that

**Comment by juga0** [comment]
Squashed and force pushed
